Documentation/WEB2WEB.md

```diff
@@ -0,0 +1,124 @@
+# Web2Web Integration
+
+Starting from version 2.2.0 of Kovalee iOS SDK, Web2Web is supported. 
+
+In order to works correctly the app needs to supports deeplinks that follow the format below (the Kovalee team will configure it for you):
+
+```
+https://appname.go.link/web2web?adj_t=123456&adj_linkme=1&user_id={{user.id}}
+```
```

Comment:
Does the partner has to do some setup on its side to support deeplink or everything is done in the SDK?
If I'm not mistaken, it has to at least modify the `plist` file?

Let's add some documentation about it. It can be a link to Adjust doc for example if we don't want to write our own.